conf/declaration.routes

```diff
@@ -117,7 +117,9 @@ POST        /nature-of-transaction               controllers.declaration.NatureO
 
 GET         /export-items                        controllers.declaration.ItemsSummaryController.displayPage()
 
-GET         /export-items/add                                  controllers.declaration.ItemsSummaryController.addItem()
+GET         /export-items/add                    controllers.declaration.ItemsSummaryController.addItem()
+
+GET         /export-items/:itemId/remove         controllers.declaration.ItemsSummaryController.removeItem(itemId)
```

Comment:
IMHO it would be 'cleaner' to use DELETE in this place.

Reply:
Browser can only send POST and GET